packages/graph-explorer/src/modules/GraphViewer/renderNode.test.ts

```diff
@@ -3,23 +3,31 @@
 // DEV NOTE: The DOMParser in happy-dom is not fully functional. Using jsdom until it works properly.
 
 import { createRandomName, createRandomColor } from "@shared/utils/testing";
-import { ICONS_CACHE, VertexIconConfig, renderNode } from "./renderNode";
+import { VertexIconConfig, renderNode } from "./renderNode";
 import { vi } from "vitest";
-
+import { QueryClient } from "@tanstack/react-query";
+
+// Mock the logger module
+vi.mock("@/utils", () => ({
+  logger: {
+    debug: vi.fn(),
+    error: vi.fn(),
+    log: vi.fn(),
+    warn: vi.fn(),
+  },
+}));
```

Comment:
You shouldn't need to mock the logger. It is mocked in the global setup:

https://github.com/aws/graph-explorer/blob/741be8d8144d15bc101857adc4367af4f5a44ddb/packages/graph-explorer/src/setupTests.ts#L85-L94

Reply:
I think the code that was here before predated that global mock, and maybe the `logger` utility.